index.html

```diff
@@ -10,56 +10,61 @@
   <body>
     <header id="nav-wrap">
       <nav id="nav">
+        <ul id="nav-links">
+          <li class="nav-link">Portfolio</li>
+          <li class="nav-link">About</li>
+          <li class="nav-link">Contact</li>
+        </ul>
         <h4 id="logo-placeholder">
           <a href="#">Welcome</a>
         </h4>
         <div id="menu-button-wrap">
           <img src="https://img.icons8.com/ios-glyphs/30/172B4D/menu--v1.png"/>
         </div>
       </nav>
-    </header>
-    <section id="heading">
-      <h1 id="main-heading">
+      <section id="heading">
+        <h1 id="main-heading">
             Hey there. I'm Ellon <br>
             <span>I'm a software developer</span>
-      </h1>
-      <div id="supporting-text">
-        <p>
+        </h1>
+        <div id="supporting-text">
+          <p>
             I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don't hestiate to contact me.
-        </p>
-      </div>
-      <ul id="social-icons">
-        <li class="icon">
-          <a href="https://github.com/Ellon-M">
-            <img src="https://img.icons8.com/windows/32/172B4D/github.png"/>
-          </a>
-        </li>
-        <li class="icon">
-          <a href="https://ellon-m.medium.com/">
-            <img src="https://img.icons8.com/glyph-neue/32/172B4D/medium-logo.png"/>
-          </a>
-        </li>
-        <li class="icon">
-          <a href="https://twitter.com/ellonm4/">
-            <img src="https://img.icons8.com/sf-ultralight-filled/32/172B4D/twitter.png"/>
-          </a>
-        </li>
-        <li class="icon">
-          <a href="https://www.linkedin.com/in/ellon-m/">
-            <img src="https://img.icons8.com/ios-filled/32/172B4D/linkedin.png"/>
-          </a> 
-        </li>
-        <li class="icon">
-          <a href="https://angel.co/u/ellon-mordecai">
-            <img src="https://img.icons8.com/ios/32/172B4D/angelist.png"/>
-          </a>
-        </li>
-      </ul>
-      <div id="main-button">
-        <img src="https://img.icons8.com/ios/45/172B4D/circled-chevron-down.png"/>
-      </div>
-    </section>
+          </p>
+        </div>
+        <ul id="social-icons">
+          <li class="icon">
+            <a href="https://github.com/Ellon-M">
+              <img src="https://img.icons8.com/windows/32/172B4D/github.png"/>
```

Comment:
- [ ] Please, remember to perform the changes [requested](https://github.com/Ellon-M/portfolio/pull/7#discussion_r1014538520) by the previous reviewer 🎲, add `alt` attribute for every `<img>` tag, it represents an alternate text when an image can't be displayed 🎃.